src/libs/reportUtils.js

```diff
@@ -58,7 +58,7 @@ function canEditReportAction(reportAction) {
     return reportAction.actorEmail === sessionEmail
         && reportAction.reportActionID
         && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT
-        && !isReportMessageAttachment(lodashGet(reportAction, ['message', 0, 'text'], ''));
+        && !isReportMessageAttachment(lodashGet(reportAction, ['message', 0, 'html'], ''));
```

Comment:
Everything else looks good, but I don't think we need this change. When a reportAction is an image attachment, its `html` will contain an image tag. However, its `text` will just be `[Attachment]`.